package.json

```diff
@@ -618,6 +618,7 @@
 		"test": "npm run compile && node ./out/test/runTest.js"
 	},
 	"devDependencies": {
+		"@types/chai": "^4.2.15",
```

Comment:
it looks like there we have a tech debt that all the npm modules are packaged to the release zip. It looks like most of the "dev dependency" packages are not needed in the publish package. Maybe we can clean it in future.